### PR TITLE
Firmware icon

### PR DESCRIPTION
**Pull request by jackpot51**
This depends on the cleanup pull request, #106

### CODE REVIEW

**Comment by brs17** [comment]
@isantop any ideas on why this might be happening? Should be using the same icon.

**Comment by isantop** [comment]
My guess is that it's because the icon is being set as a GtkWindow icon, instead of being packed into the headerbar as a GtkImage. GtkWindow Icons are small (supposed to fit in a standard-height titlebar).

File: system76-firmware-dialog [hunk 1]
```diff
@@ -584,6 +587,8 @@ def send_notification(data):
     GLib.timeout_add_seconds(NOTIFICATION_TIMEOUT, notification.close)
     mainloop.run()
 
+Gtk.Window.set_default_icon_name("system76-driver")
```
Comment (isantop) [inline system76-firmware-dialog]: I believe this is causing the low-resolution Dash Icon.

Comment (brs17) [reply system76-firmware-dialog]: Do you know which function should be used instead? `gtk_window_set_icon_list()` perhaps?

**Comment by jackpot51** [comment]
I figured out why it happens. I am closing in favor of adding the firmware interface to gnome-control-center.

I also found that *notifications have not been working*

**Pull request closed without merging**